app/src/androidTest/kotlin/org/wordpress/aztec/demo/tests/HistoryMixedTests.kt

```diff
@@ -24,7 +24,7 @@ class HistoryMixedTests : BaseHistoryTest() {
             "\t<li></li>\n" +
             "</ol>\n" +
             "\n" +
-            "<hr>\n" +
+            "<hr/>\n" +
```

Comment:
Technically there should be a space between "hr" and "/": `<hr />`. We never bothered with this since Calypso would always change it back to `<hr>`. The same thing would happen with `<br />`.

Reply:
Thanks! addressed in d8c45fea